static/src/javascripts/projects/common/modules/commercial/creatives/gu-style-comcontent.js

```diff
@@ -25,12 +25,25 @@ define([
     var GustyleComcontent = function ($adSlot, params) {
         this.$adSlot = $adSlot;
         this.params  = params;
+        this.contentPosition = {
+            bottom: 'gu-display__content-position--bottom',
+            top: 'gu-display__content-position--top'
+        };
+        this.fontSize = {
+            small: 'gu-display__content-size--small',
+            regular: 'gu-display__content-size--regular',
+            big: 'gu-display__content-size--big'
+        };
     };
 
     GustyleComcontent.prototype.create = function () {
         var externalLinkIcon = svgs('externalLink', ['gu-external-icon']),
             templateOptions = {
-                clickMacro: this.params.clickMacro,
+                articleContentColor: this.params.articleContentColor === 'white' ? 'gu-display__content-color--bright ' : 'gu-display__content-color--dark',
+                articleContentPosition: this.contentPosition[this.params.articleContentPosition],
+                articleHeaderFontSize: this.fontSize[this.params.articleHeaderFontSize],
+                articleTextFontSize: this.fontSize[this.params.articleTextFontSize],
+                brandLogoPosition: this.params.brandLogoPosition === 'bottom-right' ? 'gu-display__logo-pos--bottom-right' : 'gu-display__logo-pos--top-left',
```

Comment:
Do you even need to do this check? 'gu-display__logo-pos--' + this.params.brandLogoPosition